### PR TITLE
Use kubernetes-client/gen generators to generate python client

### PR DESCRIPTION
**Pull request by mbohlool**
ref: #243

### CODE REVIEW

File: scripts/update-client.sh [hunk 1]
```diff
@@ -40,29 +40,24 @@ pushd "${CLIENT_ROOT}" > /dev/null
...
-rm -rf "${CLIENT_ROOT}/test"
+GEN_ROOT="${TEMP_FOLDER}/gen"
+SETTING_FILE="${TEMP_FOLDER}/settings"
+echo "export KUBERNETES_BRANCH=\"$(python ${SCRIPT_ROOT}/constants.py KUBERNETES_BRANCH)\"" > $SETTING_FILE
+echo "export CLIENT_VERSION=\"$(python ${SCRIPT_ROOT}/constants.py CLIENT_VERSION)\"" >> $SETTING_FILE
+echo "export PACKAGE_NAME=\"client\"" >> $SETTING_FILE
 
-echo "--- Generating client ..."
-mvn -f "${SCRIPT_ROOT}/pom.xml" clean generate-sources -Dgenerator.spec.path="${SCRIPT_ROOT}/swagger.json" -Dgenerator.output.path="${CLIENT_ROOT}" -Dgenerator.package.name=client -D=generator.client.version=${CLIENT_VERSION}
+echo ">>> Clonging gen repo"
```
Comment (dims) [inline scripts/update-client.sh]: Cloning :)

**Comment by dims** [comment]
Looks good, just one tiny typo. Haven't tried it though. We can let this merge and then fixup any issues. Will try to run it later this week

**Comment by mbohlool** [comment]
Thanks. I also removed unused files too (the ones that duplicates gen repo like preprocessing script). It is working, but I need to update third party resources in the gen repo. I am going to move the https://github.com/kubernetes-incubator/client-python/pull/283 to gen repo.

**Comment by dims** [comment]
LGTM 👍

**Pull request merged**